src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+- Resolution
+  - After the climax the round finishes up, resolving the round for remaining players. Usually this is the last few minutes before the evacuation shuttle arrives.
+
+Issues arise when one of these stages are too long, too short or entirely skipped over. A round doesn't necessarily have to be ruined just because a stage is misplaced, however if there are no sufficient back-up scenarios to fall back on the round may end up frustrating for some amount of the round's players (e.g. stalling, dragged out fighting, antag gets found too early without getting to affect the round, no resolution so players just evacuate).
+
+This design document will try to ensure that a round with a Rev antag follows this pattern.
+
+### Revolutionaries Gamemode Goals
+
+The goals that we want to reach with the Revolutionaries are the following:
+- Revolutionaries should be applicable as its own round type and as a mid-round antag, equivalent to Nukies/LoneOp or Traitors/Mid-round Traitors. The difference should be in the number of antags and the resources available.
+- Round removal should be discouraged where possible. It should not be the optimal choice to gib or leave a player to rot unless in specific circumstances.
+- Revolutionaries should not be discovered through *no fault of their own* in the Initial Conversions phase (i.e. metagameable mechanics). 
+- A single non-Security crew member discovering an antag should not spell the end of the antag faction's progression in the round.
+- Round end conditions should be clear and straight-forward, though a crew overcoming the antag shouldn't necessarily mean the end of the round.
+- Crew shouldn't be able to hard counterplay the antag without engaging with the antag. 
+
+
+## New Revolutionaries, Basic layout
+<sup>Any values given here are suggestions and are especially marked by being **bolded**. Any such value should be adjusted based on playtesting.</sup>
+
+The Revolutionary antag begins either as a 2-4 roundstart antag roll for the Revolutionary gamemode, or a single crewmember secretly becomes a Revolutionary antag during another gamemode midround.
+
+Revolutionary antags that start out this way are considered Head Revolutionaries (HeadRevs). A Head Revolutionary has the duty to begin the conversion of other crew and have some extra tools at their disposal to strengthen the progression of the antag faction, but they are not strictly necessary for the revolution antags to succeed. The goal for Head Revolutionaries is to convert crew to grow the faction's strength, convert department Heads and survive until the evacuation shuttle without being deconverted. 
+
+Head Revolutionaries share the same team and when they are given the role they obtain the names and jobs of the other Head Revolutionaries on the station. Revolutionaries can be identified by other revolutionaries via an icon next to their character, however Head Revs do not have a unique icon.
```

Comment:
I am not opposed to making the R blue for headrevs to see each other, as headrevs get each others' names roundstart.

With how the conversion mechanics have been changed and the core problem of "one Cargo Tech gets deconverted, rats out all the revolutionary leaders" is something I want to avoid.

Head Revs have ways to reveal themselves to their revolutionaries anyhow, by either just saying it or bringing out the DDD. This puts the choice into their hands.